_docs/quickstarts/scanner/index.md

```diff
@@ -1,62 +1,57 @@
-
-## Running the scanner
-
-* Activate the virtualenv (if applicable) of your Forseti installation.
-
-* A sample rules file can be found in the samples/ directory. You should edit
-  this to fit your environment.
-
-* To see all the available commandline flags for the scanner, run:
-
-  ```sh
-  $ forseti_scanner --helpfull
-  ```
-
-  To run the scanner with a local rules file:
-
-  ```sh
-  $ forseti_scanner --rules path/to/rules.yaml
-  ```
-
-  To run the scanner with a rules file stored in GCS, e.g.
-  `gs://my-bucket-name/rules/rules.yaml`:
-
-  ```sh
-  $ forseti_scanner --rules "gs://my-bucket-name/rules/rules.yaml"
-  ```
-
-* By default, the scanner will save the csv output to a temporary
-location. Specify an output path (it can also be a GCS bucket with 
-the `gs://<bucket-name>` syntax) in order to save the csv there.
-
-  To specify the output location for the violations:
-
-  ```sh
-  $ forseti_scanner --rules "rules/rules.yaml" --output_path path/for/output/
-  ```
-
-* If you are developing a new feature or bug fix, you can also use the
-convenience [dev_scanner.sh script](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/samples/scanner/dev_scanner.sh.sample)
-to run scanner so you don't have to set PYTHONPATH and other commandline flags
+# {{ page.title }}
+
+This quickstart describes how to get started with Forseti Scanner. Forseti
+Scanner uses a JSON or YAML rules definition file to audit your Google Cloud
+Platform (GCP) resources, such as organizations or projects. After running the
+audit, Forseti Scanner outputs rule violations to Cloud SQL and optionally
+writes it to a bucket in Google Cloud Storage.
+
+Forseti Scanner is different from the Cloud Security Scanner, which does App
+Engine vulnerability scanning. Learn more about
+[Cloud Security Scanner](https://cloud.google.com/security-scanner/).
+
+## Running Forseti Scanner
+
+To run Forseti Scanner, follow the process below:
+
+  1. Activate any virtualenv you're using for your Forseti installation,
+  if applicable.
+  
+  1. Edit the sample `rules.yaml` file in the `samples/` directory to fit your
+  environment. Learn more about
+  [defining rules for Forseti Scanner]({% link _docs/howto/scanner-rules.md %}).
+  
+  1. Run Scanner for your rules file location:
+  
+     1. If you're using a local rules file, run the following command wherex
+       `RULES_PATH` is the location of your rules file:
+      
+            forseti_scanner --rules RULES_PATH.yaml
+            
+     1. If you're using a rules file stored on Google Cloud Storage, such
+      as `gs://my-project-id/rules/rules.yaml`, run the following command
+      where `RULES_PATH` is the location of your rules file in the project,
+      and `PROJECT_ID` is the project ID in which the rules file is stored:
```

Comment:
Thanks, re: more detailed instructions on running forseti_scanner -- should I do this on the CL itself, or here?